### PR TITLE
Info and procedure for bulk email change

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2965](https://openedx.atlassian.net/browse/DOC-2965)

Eric made a nice but high impact change to bulk email.
### Date Needed

within the hour if possible
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @efischer19 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check/copy edit/dev edit): @srpearce 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: @mollydb , @bderusha 
- [x] test
### Post-review
- [ ] Squash commits

### CODE REVIEW

**Comment by efischer19** [comment]
👍 Accurate and concise, looks great!

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+################################
+Enabling the Bulk Email Feature
+################################
+
+This section describes a change to the process for enabling or disabling the
+bulk email feature. This change was released to the edx-platform repository on
+24 May 2016. On that date, the bulk email feature was disabled on all Open edX
+installations. To re-enable the bulk email feature, you must follow the new
+procedure descibed in this section.
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: typo: described (could also change to "that this section describes")

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+.. contents::
+   :local:
+   :depth: 1
+
+**********************
+About this Change
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: Cap "This"

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+**********************
+
+Prior to this change, you enabled or disabled the bulk email feature by making
+manual updates to several files, including the ``lms/envs/common.py`` Python
+file. The two relevant settings in the FEATURES section of this file were
+``ENABLE_INSTRUCTOR_EMAIL`` and ``REQUIRE_COURSE_EMAIL_AUTH``.
+
+.. important:: As of 21 May 2016, ``ENABLE_INSTRUCTOR_EMAIL: False`` for
+  all Open edX sites that follow the edx-platform repository closely.
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst:27]: Is a verb missing here? E.g., "`ENABLE_INSTRUCTOR_EMAIL: False` is set for" or "`ENABLE_INSTRUCTOR_EMAIL: False` must be set for "?

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+To simplify the process of changing the bulk email settings,  a **Bulk_Email**
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: Extra space after comma

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+option is now available on the Django admin console.
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: on --> in
admin --> administration

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+**********************************************************
+Changing the Bulk Email Setting with Django Admin
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: "Django Admin" sounds slangy - I'd write this out, or leave out "with the Django Administration Console" if it's too long

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+**********************************************************
+
+To enable or disable the bulk email feature for an Open edX site, follow these
+steps.
+
+#. Sign in to the Django administration console for your base URL. For example,
+   ``http://{your_URL}/admin``.
+
+#. In the **Bulk_Email** section, next to **Bulk email flags** select **Add**.
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: add comma after "flags"

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+#. To enable the bulk email feature, select the **Enabled** check box.
+
+   To disable the bulk email feature, clear the **Enabled** check box.
+
+#. If you previously set ``REQUIRE_COURSE_EMAIL_AUTH: True``, select the
+   **Require course email auth** checkbox.
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: checkbox --> check box

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+#. Select **Save** at the bottom of the page.
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: "At the bottom of the page, select Save" (tell folks where to go before telling them what to do)

File: en_us/open_edx_release_notes/source/bulk_email.rst [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+#.
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/bulk_email.rst]: Typo?

**Comment by lamagnifica** [comment]
@srpearce please take another look if you need to while tests are running

**Comment by srpearce** [comment]
👍

**Pull request merged**